mc2/data/download_template.sql

```diff
@@ -11,7 +11,7 @@ with a1 AS (
     sum(active_hours_sum) AS usage_all,
     count(DISTINCT(client_id)) AS dau_all
   FROM
-    telemetry.clients_daily_v6 HH
+    `moz-fx-data-shared-prod`.telemetry.clients_daily_v6 HH
```

Comment:
I am fascinated by this HH, JJ, etc. naming convention. Where does that come from?

Reply:
That would be a question for @saptarshiguha or @wcbeard 😄

Reply:
I'm not sure, but it's going to disappear soon

Reply:
the recesses of my mind